### DISCUSSION TITLE
NanoELS H4 build on WM210V Platform

### DISCUSSION THREAD

**Discussion by 5280Hi** [kachurovskiy/nanoels#147] [Show and tell]
Below are photos of my H4 implementation. I am a life long woodworker, but fairly new to metal fabrication. I recently purchased a WM210V based lathe.  It became immediately obvious to me that I did not want to mess around with manual lead screw gear changes. I figured a microprocessor driven stepper motor was the way to go, and then discovered a boat load of ELS builds on YouTube. After researching many of these, I decided that Maxim's H4 design was the best match for what I wanted. Also, his design was implemented on a variant of the same lathe that I have.

My implementation is fairly close to Maxim's most recent design. The only real problem I had with the build was with the plastic spindle encoder gear. I printed several of these with various materials, but all of them made a lot of noise and had pronounced runout wobble. I solved this problem by creating a hybrid encoder gear. I turned the main body of the gear (rim) out of aluminum and printed the gear teeth (tire) out of ABS. The rim OD was made to create a press fit with the ABS tire. This solution eliminated most of the wobble and greatly reduced the gear noise.

I also designed a variant of the lead screw motor mount and cut it out of 1/2" aluminum on my CNC router. The design includes the ability to adjust the belt tension. In addition, I modified the encoder bracket to provide a split ring clamping mechanism and expanded the adjustment range. It was printed with carbon fiber nylon.

The cross slide housing was printed with high temp carbon fiber nylon and remaining parts with ABS. The cross slide was implemented using Felix Weber's elegant design. The lead screw motor cover and ELS Controller base were printed with PLA.

With all the information that was provided by Maxim and several others, My build went much smoother than I expected. Other than the noise problem I had with the encoder gear, everything worked on the first try. It certainly helps to be later in the learning curve and benefit from others improvements and debugging. In particular, I want to thank Maxim for his excellent design, documentation, and software skills, and Felix for his design that I consider to be a work of art.

The only improvement I can suggest is to provide a master index PDF with links to all of the documentation (GitHub, YouTube videos, Thingiverse designs, etc.). This would enable being able to go to a single place to quickly gain access to all available information.

![_F502469](https://github.com/kachurovskiy/nanoels/assets/153134519/287dd74f-3765-4473-aff9-dbefe6bf5e99)
![_F502468](https://github.com/kachurovskiy/nanoels/assets/153134519/6ef9b78e-eb09-4ba7-a4b1-bb6885fb0420)
![_F502458](https://github.com/kachurovskiy/nanoels/assets/153134519/f34a8d7f-7883-4fc3-a3dc-ee3072f15bc1)
![_F502460](https://github.com/kachurovskiy/nanoels/assets/153134519/6d8adf06-287a-41fb-be83-ff3454d4456b)
![_F502464](https://github.com/kachurovskiy/nanoels/assets/153134519/f9d8b6ad-7459-411a-b8ab-6bd4e967ef4e)
![_F502466](https://github.com/kachurovskiy/nanoels/assets/153134519/ee9c052e-2a18-498f-bf17-3d8a99b08441)

**Comment by kachurovskiy** [comment]
Huge thanks for sharing your build. Very well done! So white 😍

Would be really cool to see a closer picture of your encoder setup.

Documentation indeed needs more work 🙄

**Reply by 5280Hi** [reply]
I have added some detailed photos of the encoder gear and mount. As you can tell, I like white. It helps make for a light and bright shop.

![_F502473](https://github.com/kachurovskiy/nanoels/assets/153134519/c3a3ce63-5c35-47d7-ac6f-19a5029b97f6)
![_F502474](https://github.com/kachurovskiy/nanoels/assets/153134519/6d1cc17d-9f04-478e-8532-d5d8688f1035)
![_F502477](https://github.com/kachurovskiy/nanoels/assets/153134519/4a1eda25-2812-4c75-90f2-6456e6a0c9e1)
![_F502478](https://github.com/kachurovskiy/nanoels/assets/153134519/f31d4411-8920-4ab2-81b7-1de563a0fea9)
![_F502479](https://github.com/kachurovskiy/nanoels/assets/153134519/a1c63b1c-f5a8-4935-adfe-87925297fb9a)
![56 Tooth Encoder Gear Tire v2](https://github.com/kachurovskiy/nanoels/assets/153134519/5b99eee7-8542-4431-bf6b-ae5499d3aec7)

**Comment by Festivejelly** [comment]
Nice clean build!

**Comment by vegetate7** [comment]
Nice build!
But don't geared encoder have a backlash?

**Reply by Festivejelly** [reply]
I guess only if changing direction?

**Reply by 5280Hi** [reply]
Yes, and normally the spindle is only traveling in one direction. Also, Maxim has programmed backlash compensation in the software in the even of direction change..

**Reply by vegetate7** [reply]
Yep. You are right. That one small thing I completely forget :)  The spindle is (amost) all time running one direction.
